app/v1/controllers/AbbreviatedReturnController.scala

```diff
@@ -19,16 +19,16 @@ package v1.controllers
 import javax.inject.{Inject, Singleton}
 import play.api.libs.json.JsValue
 import play.api.mvc.{Action, ControllerComponents}
-import v1.controllers.actions.AuthAction
+import v1.controllers.actions.{AuthActionProvider}
```

Comment:
I am right in assuming that the Braces here don't actually do anything?
unless you added in some other individual elements from v1.controllers.actions.{x,y,z}?

Reply:
yup - you are right!